tests/matrix_free/parallel_multigrid_adaptive_03.cc

```diff
@@ -576,7 +576,8 @@ void test ()
 
 int main (int argc, char **argv)
 {
-  Utilities::MPI::MPI_InitFinalize mpi_init(argc, argv, 1);
+  Utilities::MPI::MPI_InitFinalize mpi_init(argc, argv,
+                                            numbers::invalid_unsigned_int);
```

Comment:
Please use `testing_max_num_threads()` instead of `numbers::invalid_unsigned_int` to not create too many threads.